homepage.php

```diff
@@ -49,57 +49,57 @@
 		<div class="content_4">
 			<Space></Space>
 			<div class="images1">
-				<a href = "Query8.php"><img src = "images/indie.jpg" class="image1"></a>
+				<a href = "Query8.php"><img src = "images/Indie2.jpg" class="image1"></a>
 				<!--Div for overlay techniques so that when the user hover it hide the images and display text -->
 				<div class="overlay">
     				<div class="text"><a href = "Query8.php">Indie</a></div>
   				</div>
 			</div>
 			<div class="images2">
-				<a href = "Query7.php"><img src = "images/R&B.jpg" class="image2"></a>
+				<a href = "Query7.php"><img src = "images/RandB2.jpg" class="image2"></a>
 				<!--Div for overlay techniques so that when the user hover it hide the images and display text -->
 				<div class="overlay">
     				<div class="text"><a href = "Query7.php">R and B</a></div>
   				</div>
 			</div>
 			<div class="images3">
-				<a href = "Query6.php"><img src = "images/pop.jpg" class="image3"></a>
+				<a href = "Query6.php"><img src = "images/Pop2.png" class="image3"></a>
 				<!--Div for overlay techniques so that when the user hover it hide the images and display text -->
 				<div class="overlay">
     				<div class="text"><a href = "Query6.php">Pop</a></div>
   				</div>
 			</div>
 			<div class="images4">
-				<a href = "Query10.php"><img src = "images/rock.png" class="image4"></a>
+				<a href = "Query10.php"><img src = "images/rock2.png" class="image4"></a>
 				<!--Div for overlay techniques so that when the user hover it hide the images and display text -->
 				<div class="overlay">
     				<div class="text"><a href = "Query10.php">Rock</a></div>
   				</div>
 			</div>
 			<Space></Space>
 			<div class="images5">
-				<a href = "Query9.php"><img src = "images/easylisten.jpg" class="image5"></a>
+				<a href = "Query9.php"><img src = "images/5.jpg" class="image5"></a>
 				<!--Div for overlay techniques so that when the user hover it hide the images and display text -->
 				<div class="overlay">
     				<div class="text"><a href = "Query9.php">Easy Listen</a></div>
   				</div>
 			</div>
 			<div class="images6">
-				<a href = "Query3.php"><img src = "images/country.png" class="image6"></a>
+				<a href = "Query3.php"><img src = "images/6.jpg" class="image6"></a>
```

Comment:
30/8/2021: I trailing for some img for it is more match so it could use as an indicator to show the user that the img is the link to a query that contain all the song that is like the img represent. Also decided a nice img so it can improve the aesthetic of the img.